evap/grades/templates/grades_course_base.html

```diff
@@ -1,11 +1,12 @@
 {% extends "grades_semester_base.html" %}
 
 {% load i18n %}
+{% load morefilters %}
 
 {% block subtitle %}
     {{ block.super }}
     {% if course %}
-    	{% if disable_breadcrumb_course %}
+    	{% if disable_breadcrumb_course or is_grade_publisher|is_false %}
```

Comment:
no that is checking whether it's false. if it's not existing this will not return true:
```
@register.filter
def is_false(arg):
    return arg is False
```